database/build.go

```diff
@@ -75,6 +77,13 @@ func (b *Build) Crop() *Build {
 		b.Message = sql.NullString{String: b.Message.String[:maxMessageLength], Valid: true}
 	}
 
+	// trim the Error field to 500 characters — 250 from each end to maximize usefulness.
+	if len(b.Error.String) > maxErrorLength {
+		front := maxErrorLength - (maxErrorLength / 2)
+		end := len(b.Error.String) - (maxErrorLength / 2)
+		str := b.Error.String[:front] + b.Error.String[end:]
+		b.Error = sql.NullString{String: str, Valid: true}
+	}
```

Comment:
Are we sure we're taking the right approach here?

Personally, I'd rather start with increasing the number of characters before going down this route.

If we feel that's insufficient, then we should use compression for the `error` field.

Reply:
I understand not wanting to process string logic like this, but I am curious as to why the `message` and `title` fields get cropped in a similar fashion? 

Also, I do wonder if it may be more efficient to handle larger errors by referencing the log from which it came rather than attempt to store some truncated version.